layouts/index.html

```diff
@@ -50,11 +46,25 @@ <h1 class="usa-display">Register to vote</h1>
 
     </div>
 
-    {{/*
     {{ partial "analytics.html" . }}
+
+    {{/*
     {{ partial "scripts.html" . }}
     */}}
 
+    <script>
+      // TODO: This exists only for basic testing. It will be removed once
+      // the designs are more fleshed out. This is just a work in progress for now.
+      ( function( g ) {;
+        var button = document.getElementById( 'js-user-submit' );
+        button.addEventListener( 'click', function ( event ) {
+          event.preventDefault();
+          var selection = document.getElementById( 'js-user-selection' );
+          g.location.href = g.location.origin + '/states/' + selection.value;
+        }, false );
+      } )( this );
+    </script>
+
```

Comment:
This is just a placeholder to get this MVP working locally. I will be removing this functionality as we flesh out the `autocomplete` / `dropdown` search navigation on the homepage.